lib/utils/FileCache.mjs

```diff
@@ -60,23 +60,54 @@ export default class FileCache {
 		// Get file descriptor compares current metadata against cached
 		// one and stores the result to "changed" prop.w
 		const descriptor = this._fileCache.getFileDescriptor(absoluteFilepath);
-		/** @type {CacheMetadata} */
-		const meta = descriptor.meta || {};
-		const changed = descriptor.changed || meta.hashOfConfig !== this._hashOfConfig;
+
+		descriptor.meta.data ??= {};
```

Comment:
See: https://github.com/jaredwray/cacheable/tree/5ec1858bdd07aa5efc30e87f14253a951fbc83b5/packages/file-entry-cache#changes-from-v9-to-v10

> On `FileEntryDescriptor.meta` if using typescript you need to use the `meta.data` to set additional information. This is to allow for better type checking and to avoid conflicts with the meta object which was any.